db/schema.rb

```diff
@@ -329,22 +329,24 @@
     t.string "purchid"
     t.integer "type"
     t.integer "ownershipsch"
-    t.string "othtype"
-    t.integer "jointmore"
     t.integer "jointpur"
+    t.string "othtype"
     t.integer "beds"
-    t.integer "companybuy"
+    t.integer "jointmore"
     t.integer "age1"
     t.integer "age1_known"
     t.string "sex1"
+    t.integer "national"
+    t.string "othernational"
     t.integer "buy1livein"
+    t.integer "companybuy"
     t.integer "buylivein"
     t.integer "builtype"
     t.integer "proptype"
-    t.string "relat2"
-    t.string "otherrelat2"
     t.integer "age2"
     t.integer "age2_known"
+    t.string "relat2"
+    t.string "otherrelat2"
     t.integer "ethnic"
```

Comment:
I'm not sure how to avoid these reordering - I think it's because of the migrations added since this branch was made. I've done a rebase but that didn't seem to help. I know it's not an issue really but if schema order is something we should keep consistent how do I do that?

Reply:
If it's just re-ordering, I wouldn't worry too much about it.